language-support/ts/codegen/tests/ts/build-and-lint-test/package.json

```diff
@@ -7,6 +7,7 @@
   "dependencies": {
     "@daml/ledger": "0.0.0",
     "@daml/types": "0.0.0",
+    "@daml.js/build-and-lint-1.0.0": "file:../daml2ts/build-and-lint-1.0.0",
```

Comment:
Why didn't we depend on that package before?

Reply:
it's a good question. we just didn't but it didn't matter because the topological ordering of the workspaces meant it was in node_modules before we came to build it.